kornia/geometry/liegroup/se2.py

```diff
@@ -0,0 +1,235 @@
+# kornia.geometry.se2 module inspired by Sophus-sympy.
+# https://github.com/strasdat/Sophus/blob/master/sympy/sophus/se2.py
+from typing import Optional, Tuple
+
+from kornia.core import Module, Parameter, Tensor, concatenate, pad, rand, stack, tensor, where
+from kornia.geometry.liegroup._utils import check_se2_r_t_shape, check_v_shape
+from kornia.geometry.liegroup.so2 import So2
+from kornia.testing import KORNIA_CHECK, KORNIA_CHECK_TYPE
+
+
+class Se2(Module):
+    r"""Base class to represent the Se2 group.
+
+    The SE(2) is the group of rigid body transformations about the origin of two-dimensional Euclidean
+    space :math:`R^2` under the operation of composition.
+    See more:
+
+    Example:
+        >>> so2 = So2.identity()
+        >>> t = torch.ones((2))
+        >>> se2 = Se2(so2, t)
+        >>> se2
+        rotation: (1+0j)
+        translation: Parameter containing:
+        tensor([1., 1.], requires_grad=True)
+    """
+
+    def __init__(self, r: So2, t: Tensor) -> None:
```

Comment:
```suggestion
    def __init__(self, rotation: So2, translation: Tensor) -> None:
```
can we start adding more explicit names ? eventually we can add aliases as properties